src/otio_aaf_adapter/adapters/advanced_authoring_format.py

```diff
@@ -1638,27 +1663,15 @@ def read_from_file(
     return result
 
 
-def write_to_file(
-    input_otio,
-    filepath,
-    prefer_file_mob_id=False,
-    use_empty_mob_ids=False,
-    **kwargs
-):
+def write_to_file(input_otio, filepath, **kwargs):
 
     with aaf2.open(filepath, "w") as f:
 
         timeline = aaf_writer._stackify_nested_groups(input_otio)
 
         aaf_writer.validate_metadata(timeline)
 
-        otio2aaf = aaf_writer.AAFFileTranscriber(
-            timeline,
-            f,
-            prefer_file_mob_id=prefer_file_mob_id,
-            use_empty_mob_ids=use_empty_mob_ids,
-            **kwargs
-        )
+        otio2aaf = aaf_writer.AAFFileTranscriber(timeline, f, **kwargs)
```

Comment:
Same: This undoes the work recently merged in #56.

Reply:
Oops. Fixed.